src/main/java/mediathek/tool/MVSubtitle.java

```diff
@@ -131,7 +139,8 @@ public static void writeSubtitle( DatenDownload datenDownload) {
 
         try (TimedTextMarkupLanguageParser ttmlp = new TimedTextMarkupLanguageParser()) {
             if (strSubtitelFile != null) {
-                if (!strSubtitelFile.endsWith('.' + SUFFIX_SRT)) {
+                if (!strSubtitelFile.endsWith('.' + SUFFIX_SRT) && !strSubtitelFile.endsWith("." + SUFFIX_VTT)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)